app/assets/stylesheets/mixins/vars.sass

```diff
@@ -70,6 +70,11 @@ $color-button-blue: #ebf4f3
 $color-button-blue-dark: #c5e0dc
 $color-button-video: #dcdcdc
 
+$color-icon-facebook: #3b5988
+$color-icon-twitter: #1da1f2
+$color-icon-linkedin: #0077b5
+$color-icon-flipboard: #e12828

```

Comment:
With the new icons I've added you won't need to set the colours manually any more, don't forget to remove these.